apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console.Tests/GoogleCloudConsoleFormatterTest.cs

```diff
@@ -152,6 +153,101 @@ public void ConsoleLoggerOptions_OptionChange_IsReloaded()
         Assert.Equal(expectedJsonWithScope, actualJsonWithScope);
     }
 
+    [Fact]
+    public void ConsoleLoggerOptions_EmptyProjectId_TraceInformationIsNotLogged()
```

Comment:
We're not starting an activity in this test, so presumably it wouldn't have anything to log anyway... I think we should start an activity here - basically the only difference between this and the test that *does* log trace information should be the empty project ID.

Reply:
+1